### DISCUSSION TITLE
Style not working

### DISCUSSION THREAD

**Discussion by pengpeng0119** [storybookjs/storybook#28732] [Help]
### Summary

What happened is that I repackaged quasar into my commonly used components, and now I want to document the repackaged components, but all the styles of quasar do not take effect in the storybook. 

### Additional information

![image](https://github.com/user-attachments/assets/aef71ebb-88e9-494c-ab9e-7e77395e055b)

**Comment by jjoshi-space** [comment]
@pengpeng0119 Is it that your index.sass is not imported correctly?
Also, if you get a chance, can you check this discussion out? https://github.com/storybookjs/storybook/discussions/28713
I wonder if you as well came across loading scss variables in your quasar config

Thanks
